applications/tap/secrets-idfint.yaml

```
@@ -1,3 +1,6 @@
 uws-db-password:
   description: >-
     Password for external UWS PostgreSQL server
+qserv-password:
+  description: >-
+    Password for the QServ database server
```

You need the conditional here so that the secret isn't required everywhere.
```suggestion
qserv-password:
  description: >-
    Password for the QServ database server
  if: cadc-tap.config.qserv.passwordEnabled
```